{{ cookiecutter.folder_name }}/{{ cookiecutter.import_name }}/bin/main.py

```diff
@@ -0,0 +1,98 @@
+"""
+`{{cookiecutter.import_name}}` is the top-level command for accessing various subcommands.
+
+Try:
```

Comment:
Is this a hanging docstring?

Reply:
It's auto-generated based on the available subcommands (the `--help` stuff)